remote/remote_state.go

```diff
@@ -112,7 +113,8 @@ func (remoteState *RemoteState) differsFrom(existingBackend *TerraformBackend, t
 	}
 
 	if !terraformStateConfigEqual(existingBackend.Config, remoteState.Config) {
-		terragruntOptions.Logger.Printf("Backend config has changed from %s to %s", existingBackend.Config, remoteState.Config)
+		terragruntOptions.Logger.Printf("Backend config has changed")
```

Comment:
NIT: perhaps mention in this log statement that you can run with the debug flag / env var to see the details of what changed?

Reply:
Done: 760d5ca